### DISCUSSION TITLE
Delayed on getting education benefits!

### DISCUSSION THREAD

**Discussion by Manojps122** [community/community#140184] [GitHub Education]
### Hi everyone,

its been 3days i have not yet recieved the github benefits after applying for it can u please solve the issue

**Comment by davevad93** [comment]
Hi @Manojps122 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Manojps122 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
